clang/test/CodeGen/sanitize-coverage-old-pm.c

```diff
@@ -9,7 +9,6 @@
 // UNSUPPORTED: armv7, armv7l, thumbv7, armv8l
 // The same issue also occurs on a riscv32 host.
 // XFAIL: riscv32, nanomips
```

Comment:
I think UNSUPPORTED is the correct choice here.
XFAIL is for documenting known bugs or issues.